addons/launcher-template/launcher-template.addon

```diff
@@ -0,0 +1,4 @@
+# Name: launcher-template
```

Comment:
The official name is launch.openshift.io, but I couldn't use it as the addon name, so I adopted as "launcher" instead